src/features/elementfeature.jl

```diff
@@ -40,48 +41,71 @@ function ElementFeature(
     else
         vector_length = nbins
     end
-    encode_f =
-        atoms -> reduce(
-            hcat,
+
```

Comment:
this constructor looks good generally...could you add a line to just crop out unnecessary columns in `lookup_table`? Since there may eventually be a fair number of these objects floating around, I think that memory savings makes sense. And very often, `lookup_table` will be read out of some big CSV that has a ton of different columns for various FD's, but a single FD object should only need the columns for elemental symbol and that FD's encoded values.

Reply:
Resolved in 399682d.